content/support/_index.md

```diff
@@ -0,0 +1,78 @@
+image = "2022/06/01hb-4958-4rxz-4t0q.png"
+hed = "Support Our State College Bureau"
+body = "Want to support our investigative journalism about Penn State and our State College regional bureau?"
+actionUrl = "https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&theme=centrecounty&campaign=7015G0000013pZTQAY"
+actionText = "Support the Bureau →"
+url = "/support/state-college"
+
+[[blocks]]
+layout = "text-3"
+title = "Who Supports Us?"
+show_partners = 4
+copy = """
+
+  <p class="md:leading-relaxed">Spotlight PA’s unique model of public-service journalism relies on the generous support of individuals and institutions who understand the vital importance of Spotlight PA’s work to the future of Pennsylvania. The newsroom is supported by a dedicated group of local and national foundations, media partners, individual major donors (“Leaders in Action”), and thousands of readers from across Pennsylvania (“members”). Consistent with our mission, we disclose the source of every dollar we receive, and we do not accept anonymous gifts.</p>
+  
+  ## Learn more about who supports our work
+
+  [Institutional funders and members since launch →](/support/funders-and-members#major-donors-and-funders)
+
+  [Leaders in Action →](/support/leaders-in-action/#our-current-leaders)
+
+  [State College regional bureau supporters →](/support/state-college/#state-college-bureau-donors)
+"""
+
+[[blocks]]
+title = "Transparency"
+layout = "text-2"
+copy="""
+  {{<featured/large>}}Spotlight PA is a nonpartisan, independent newsroom paid for by grants from foundations and donations from individuals committed to robust investigative reporting. Spotlight PA editors and reporters operate independently of our funders and maintain editorial control over all of the content they produce. Funders do not have input into the selection of which stories Spotlight PA pursues or the reporting process for those stories. Funders do not review any content before publication or have any special access to reporters or newsroom leadership. In short, we answer to you, the public.{{</featured/large>}}
+
+  {{<featured/large>}}Funds for Spotlight PA are donated to the nonprofit <a href="https://www.lenfestinstitute.org/" class="text-yellow">Lenfest Institute for Journalism</a>, which then directs them to our work. As part of our gift acceptance policy, and consistent with our promise to be fully transparent, we do not accept anonymous donations.{{</featured/large>}}
+"""
++++
+{{<featured/large>}}Spotlight PA produces investigative and public-service journalism that gets results, and shares that work at no cost with communities across Pennsylvania. But we can't do it without your support. Here are the several ways you can sustain our newsroom with a tax-deductible donation today:{{</featured/large>}} 
+
+{{<about/action-link href="https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign=7015G000001Z88SQAS">}}Donate With Credit Card  →{{</about/action-link>}}
```

Comment:
For analytics purposes, I like all the outbound links to be to /donate/, which gets reported to Google Analytics and then translated into a specific https://checkout.fundjournalism.org URL based on the attributes of the page. Is 7015G000001Z88SQAS a special campaign ID or just a generic one?